<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PriemniciKorekcija</class>
 <widget class="QWidget" name="PriemniciKorekcija">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>987</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>1198</width>
     <height>491</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="3">
     <widget class="QLabel" name="label_39">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#0000ff;&quot;&gt;Приемница - Калкулација - Корекција&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_53">
      <property name="text">
       <string>Бр на Приемница</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_25">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Датум на приемница</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QDateTimeEdit" name="dateTimeEdit">
      <property name="date">
       <date>
        <year>2000</year>
        <month>3</month>
        <day>1</day>
       </date>
      </property>
      <property name="displayFormat">
       <string>d/M/yyyy</string>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Шифра коминт.</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_27">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>Коминтент</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3" colspan="2">
     <widget class="QLineEdit" name="lineEdit">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="9">
     <widget class="QPushButton" name="pushButton_4">
      <property name="text">
       <string>Потврди</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/sterna/Resources/key_f9.png</normaloff>:/sterna/Resources/key_f9.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_45">
      <property name="text">
       <string>Документ бр.</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_14"/>
    </item>
    <item row="3" column="2">
     <widget class="QLabel" name="label_46">
      <property name="text">
       <string>Документ вид.</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QComboBox" name="comboBox_2">
      <property name="editable">
       <bool>true</bool>
      </property>
      <item>
       <property name="text">
        <string>Фактура</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Испратница</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Сметка</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="3" column="6">
     <widget class="QLabel" name="label_51">
      <property name="text">
       <string>Датум.док.</string>
      </property>
     </widget>
    </item>
    <item row="3" column="7">
     <widget class="QDateTimeEdit" name="dateTimeEdit_2">
      <property name="date">
       <date>
        <year>2000</year>
        <month>3</month>
        <day>1</day>
       </date>
      </property>
      <property name="displayFormat">
       <string>d/M/yyyy</string>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="9">
     <widget class="QPushButton" name="pushButton_5">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_55">
      <property name="text">
       <string>Валута</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QDateTimeEdit" name="dateTimeEdit_4">
      <property name="date">
       <date>
        <year>2000</year>
        <month>3</month>
        <day>1</day>
       </date>
      </property>
      <property name="displayFormat">
       <string>d/M/yyyy</string>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLabel" name="label_41">
      <property name="text">
       <string>Износ со ДДВ</string>
      </property>
     </widget>
    </item>
    <item row="4" column="6">
     <widget class="QLabel" name="label_52">
      <property name="text">
       <string>Рабат (%)</string>
      </property>
     </widget>
    </item>
    <item row="4" column="7">
     <widget class="QLineEdit" name="lineEdit_16">
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="9">
     <widget class="QPushButton" name="pushButton_6">
      <property name="text">
       <string>Бриши</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_49">
      <property name="text">
       <string>Пресметан Износ</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="lineEdit_6">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QLabel" name="label_48">
      <property name="text">
       <string>Разлика</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QLineEdit" name="lineEdit_7">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="4">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Излезна Вредност</string>
      </property>
     </widget>
    </item>
    <item row="5" column="5">
     <widget class="QLineEdit" name="lineEdit_10">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="6">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Добивка вк.</string>
      </property>
     </widget>
    </item>
    <item row="5" column="7">
     <widget class="QLineEdit" name="lineEdit_11">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="9">
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>F12 - Додај </string>
      </property>
      <property name="shortcut">
       <string>F12</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Шифра артикал</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="lineEdit_12">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QLabel" name="label_47">
      <property name="text">
       <string>Артикал</string>
      </property>
     </widget>
    </item>
    <item row="6" column="6">
     <widget class="QLabel" name="label_50">
      <property name="text">
       <string>Едм</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="6" column="7">
     <widget class="QLineEdit" name="lineEdit_8">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="6" column="8">
     <widget class="QLabel" name="label_54">
      <property name="text">
       <string>Рок Траење</string>
      </property>
     </widget>
    </item>
    <item row="6" column="9">
     <widget class="QDateTimeEdit" name="dateTimeEdit_3">
      <property name="date">
       <date>
        <year>2000</year>
        <month>3</month>
        <day>1</day>
       </date>
      </property>
      <property name="displayFormat">
       <string>d/M/yyyy</string>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_42">
      <property name="text">
       <string>Количина</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="lineEdit_4"/>
    </item>
    <item row="7" column="2">
     <widget class="QLabel" name="label_21">
      <property name="text">
       <string>Наб.Цена без ДДВ</string>
      </property>
     </widget>
    </item>
    <item row="7" column="3" colspan="2">
     <widget class="QLineEdit" name="lineEdit_3"/>
    </item>
    <item row="7" column="8">
     <widget class="QLabel" name="label_56">
      <property name="text">
       <string>Залиха</string>
      </property>
     </widget>
    </item>
    <item row="7" column="9">
     <widget class="QLineEdit" name="lineEdit_26">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_43">
      <property name="text">
       <string>Прен.ДДВ(%)</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLineEdit" name="lineEdit_9">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="8" column="2">
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>ДДВ (ден)</string>
      </property>
     </widget>
    </item>
    <item row="8" column="3" colspan="2">
     <widget class="QLineEdit" name="lineEdit_13">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="8" column="8">
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>Забелешка</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label_38">
      <property name="text">
       <string>Рабат(%)</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QLineEdit" name="lineEdit_17"/>
    </item>
    <item row="9" column="2">
     <widget class="QLabel" name="label_34">
      <property name="text">
       <string>Вк набав. со ДДВ</string>
      </property>
     </widget>
    </item>
    <item row="9" column="3" colspan="2">
     <widget class="QLineEdit" name="lineEdit_22">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="9" column="8" rowspan="3" colspan="2">
     <widget class="QTextEdit" name="textEdit"/>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="label_37">
      <property name="text">
       <string>Маржа  (%)</string>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QLineEdit" name="lineEdit_23"/>
    </item>
    <item row="10" column="2">
     <widget class="QLabel" name="label_35">
      <property name="text">
       <string>Маржа  (ден)</string>
      </property>
     </widget>
    </item>
    <item row="10" column="3" colspan="2">
     <widget class="QLineEdit" name="lineEdit_18">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="10" column="5">
     <widget class="QLabel" name="label_32">
      <property name="text">
       <string>Прод.Цена со ДДВ</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="10" column="6" colspan="2">
     <widget class="QLineEdit" name="lineEdit_21">
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QLabel" name="label_36">
      <property name="text">
       <string>Прод.Цена без ДДВ</string>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QLineEdit" name="lineEdit_19">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="11" column="2">
     <widget class="QLabel" name="label_33">
      <property name="text">
       <string>Пресм. ДДВ</string>
      </property>
     </widget>
    </item>
    <item row="11" column="3" colspan="2">
     <widget class="QLineEdit" name="lineEdit_20">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="11" column="5">
     <widget class="QLabel" name="label_40">
      <property name="text">
       <string>Вк.Прод. со ДДВ</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="11" column="6" colspan="2">
     <widget class="QLineEdit" name="lineEdit_24">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="12" column="0" colspan="10">
     <widget class="QTableView" name="tableView">
      <property name="palette">
       <palette>
        <active>
         <colorrole role="AlternateBase">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>215</red>
            <green>215</green>
            <blue>215</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="AlternateBase">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>215</red>
            <green>215</green>
            <blue>215</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="AlternateBase">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>215</red>
            <green>215</green>
            <blue>215</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="6" column="3" colspan="3">
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="3" colspan="3">
     <widget class="QLineEdit" name="lineEdit_5"/>
    </item>
    <item row="7" column="5">
     <widget class="QLabel" name="label_20">
      <property name="text">
       <string>Наб.Цена со ДДВ</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="7" column="6" colspan="2">
     <widget class="QLineEdit" name="lineEdit_15">
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_25</tabstop>
  <tabstop>dateTimeEdit</tabstop>
  <tabstop>lineEdit_27</tabstop>
  <tabstop>lineEdit</tabstop>
  <tabstop>lineEdit_14</tabstop>
  <tabstop>comboBox_2</tabstop>
  <tabstop>dateTimeEdit_2</tabstop>
  <tabstop>dateTimeEdit_4</tabstop>
  <tabstop>lineEdit_5</tabstop>
  <tabstop>lineEdit_16</tabstop>
  <tabstop>lineEdit_6</tabstop>
  <tabstop>lineEdit_7</tabstop>
  <tabstop>lineEdit_10</tabstop>
  <tabstop>lineEdit_11</tabstop>
  <tabstop>lineEdit_12</tabstop>
  <tabstop>lineEdit_2</tabstop>
  <tabstop>lineEdit_8</tabstop>
  <tabstop>dateTimeEdit_3</tabstop>
  <tabstop>lineEdit_4</tabstop>
  <tabstop>lineEdit_3</tabstop>
  <tabstop>lineEdit_15</tabstop>
  <tabstop>lineEdit_26</tabstop>
  <tabstop>lineEdit_9</tabstop>
  <tabstop>lineEdit_13</tabstop>
  <tabstop>lineEdit_17</tabstop>
  <tabstop>lineEdit_22</tabstop>
  <tabstop>lineEdit_23</tabstop>
  <tabstop>lineEdit_18</tabstop>
  <tabstop>lineEdit_21</tabstop>
  <tabstop>lineEdit_19</tabstop>
  <tabstop>lineEdit_20</tabstop>
  <tabstop>lineEdit_24</tabstop>
  <tabstop>textEdit</tabstop>
  <tabstop>tableView</tabstop>
  <tabstop>pushButton_4</tabstop>
  <tabstop>pushButton_5</tabstop>
  <tabstop>pushButton_6</tabstop>
  <tabstop>pushButton_3</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
